witnet/prices/s/SysUsdtPrice.js

```diff
@@ -83,14 +83,14 @@ const request = new Witnet.Request()
   .addSource(binance)  
   .addSource(bitget)
   .addSource(bkex)
-  .addSource(digifinex)
+  // .addSource(digifinex)
```

Comment:
Was this on purpose?

Reply:
just to remember that sys/usdt once relayed on digifinex, as it could eventually support sys/usdt again

Reply:
But how come is this change in this PR?

Reply:
because the failing sources were detecting when passing the tests in this PR ... aparently the githubs actions that were created are either not working, or their notifications going astray